doc/index.rst

```diff
@@ -2,22 +2,22 @@ Behatch contexts
 ================
 
 .. image:: https://travis-ci.org/Behatch/contexts.svg?branch=master
-    :target: https://travis-ci.org/Behatch/contexts
```

Comment:
You have broken the images: https://github.com/GabrielCW/contexts/blob/fix-typos/doc/index.rst